visualizer/exampleGenerationRequests.js

```diff
@@ -92,7 +92,7 @@
     });
 
     if (startRule !== null) {
-      var option = Array.from(dropdown.options).find(
+      var option = Array.prototype.slice.call(dropdown.options).find(
```

Comment:
Oh, this is actually making an unnecessary copy -- just use `Array.prototype.find.call(dropdown.options, function(option) { ... })`